src/transformers/utils/quantization_config.py

```diff
@@ -418,3 +423,51 @@ def post_init(self):
                     f"""dataset needs to be either a list of string or a value in
                     ['wikitext2','c4','c4-new','ptb','ptb-new'], but we found {self.dataset}"""
                 )
+        if self.bits == 4:
+            if self.disable_exllama is not None:
+                logger.warning(
+                    "Using `disable_exllama` is deprecated and will be removed in future version. Use `use_exllama` instead."
```

Comment:
A specific version should be specified here. We typically say 2 versions after the release this will be included in

Reply:
sounds good